exchanges/okgroup/okgroup_websocket.go

```diff
@@ -504,23 +636,27 @@ func (o *OKGroup) WsProcessUpdateOrderbook(wsEventData *WebsocketDataWrapper, in
 	if err != nil {
 		log.Error(log.ExchangeSys, err)
 	}
-	updatedOb := o.Websocket.Orderbook.GetOrderbook(instrument, asset.Spot)
+	updatedOb := o.Websocket.Orderbook.GetOrderbook(instrument, a)
 	checksum := o.CalculateUpdateOrderbookChecksum(updatedOb)
 	if checksum == wsEventData.Checksum {
 		if o.Verbose {
 			log.Debug(log.ExchangeSys, "Orderbook valid")
```

Comment:
This is out of scope, but it would be nice to add an exchange name here to make this message less ambiguous